src/cdk/testing/tests/protractor.e2e.spec.ts

```diff
@@ -237,6 +241,43 @@ describe('ProtractorHarnessEnvironment', () => {
         .toBe(await browser.driver.switchTo().activeElement().getAttribute('id'));
     });
 
+    it('should be able to send key to a contenteditable', async () => {
+      const editable = await harness.editable();
+      const editableInputEvent = await harness.editableInputEvent();
+
+      await editable.sendKeys('Yi');
+
+      expect(await editable.text()).toBe('Yi');
+      expect(await editableInputEvent.text()).toBe('Count: 2');
+
+      await editable.clear();
+
+      expect(await editable.text()).toBe('');
+      // Doesn't change input event count on clear. Should it?
+      expect(await editableInputEvent.text()).toBe('Count: 2');
+    });
+
+    it('should be able to send key to a child of a contenteditable', async () => {
+      const editableP = await harness.editableP();
+
+      // WebDriver is saying that the element is not interactable.
```

Comment:
maybe webdriver expects you to click in the element first?

Reply:
Seems like this is gone, and it now only complains when the element is truly not interactable, which most probably is expected.
I may have to copy the next test in testbed.spec.ts and protractor.spec.ts instead of cross-environment.spec.ts